ksql-engine/src/main/java/io/confluent/ksql/structured/SqlPredicate.java

```diff
@@ -132,6 +137,7 @@ Predicate getPredicate() {
         }
         return (Boolean) ee.evaluate(values);
       } catch (final Exception e) {
```

Comment:
nit since not directly related to your changes, but this entire try-catch block is duplicated: could pull it out into its own method, then the `logProcessingError()` helper method would not be needed (though maybe it'd still be preferred?).

Reply:
Yeah I noticed the same thing. Wanted to leave that out of this PR though.

Reply:
I filed an issue so I dont forget: https://github.com/confluentinc/ksql/issues/2398